pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -38,12 +29,12 @@ let
 
     extraPaths = [ ];
 
-    nativeBuildInputs = [
+    nativeBuildInputs = with buildPackages.netbsd; [
       bsdSetupHook netbsdSetupHook
-      makeMinimal install tsort lorder mandoc groff statHook
+      makeMinimal
+      install tsort lorder mandoc groff statHook
```

Comment:
A bit of a nitpick, but this is a bit of a scoping mess with thing explicitly coming from `buildPackages.netbsd` and stuff like `mandoc` coming from the expressions inputs (and thus implicitly from `buildPackages`).

 This is mostly about readabilty, I was a bit confused about this for half a minute.

Also applies to FreeBSD's `mkDerivation`.